resources/js/components/catalog-new/NewCustomSelect.vue

```diff
@@ -58,16 +61,17 @@ export default {
         name: String,
         placeholder: String,
         active: Boolean,
+        openedFilter: String,
+        selectedValues: Array,
+        filter: Array,
+        clearFilterSelection: Function,
+        closeOpenedFilter: Function,
+        toggleSelect: Function,
+        handleMultiSelectChange: Function,
```

Comment:
This is a lot of props... do we really need all of them? They also could be named differently -- so that they make sense in the context of this component.

```suggestion
        name: String,
        placeholder: String,
        active: Boolean,
        open: Boolean,
        selectedOptions: Array, // or integrate into `options`
        options: Array,
        // clearFilterSelection: Function, -- move to event
        // closeOpenedFilter: Function, -- move to event
        // toggleSelect: Function, -- move to event?
        // handleMultiSelectChange: Function, move to @change event
```